### ISSUE TITLE
OLH theme: box links used in lists of articles are not accessible

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4220]
When navigating, for example an [OLH homepage](https://www.iastatedigitalpress.com/mmb/), by headings with a screenreader, the article headings are not described as links, but can be followed as if they are links to that article. When navigating by link, the link appears before the heading and is only described by the article number.

What seems to be happening is that the link itself is a box behind the other content, such that any content within that area of the screen then can be followed as if it is a link, but is not identified as a link.

Two examples of this were identified from screen reader testing #4194 
- Sample 5.1, item 11. when navigating by headings, the article headings are not described as links, but can be followed as if they are links to that article. When navigating by link, the link appears before the heading and is only described by the article number.
- Sample 5.1, item 22. popular article links are also just the article number, same behaviour as above with other articles in regards to the article title not being a link, but acting as if it were a link.

**Comment by joemull** [comment]
The use of box-link in the [markup for this block](https://github.com/BirkbeckCTP/janeway/blob/master/src/themes/OLH/templates/elements/journal/box_article.html) is causing the issue. We plan to revise this markup so the anchor properly wraps the content and make sure it works as a contextual link. Then it will go back to Steph for testing.
